### PR TITLE
harbour: fix build on Linux

### PR DESCRIPTION
**Pull request by danielnachun**
- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ ] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Comment by danielnachun** [comment]
Assuming this works in CI, I'm going to upstream the flat namespace fix for macOS as this is **not** because of the usual libtool bug, and there is an active GitHub repo to submit a PR to.

File: Formula/liblzf.rb [hunk 1]
```diff
@@ -22,6 +22,8 @@ class Liblzf < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "04201793eebd3ab8a64694c1422e4d5801be13fd26d1ed257b6c31d4a231124c"
   end
 
+  conflicts_with "harbour", because: "both install `liblzf.a` static libraries"
```
Comment (cho-m) [inline Formula/liblzf.rb:25]: May want to add conflicts in separate `CI-syntax-only` PR to avoid rebuilding bottles.

Comment (cho-m) [inline Formula/liblzf.rb:25]: May want to add conflicts in separate `CI-syntax-only` PR to avoid rebuilding bottles.

Comment (SMillerDev) [reply Formula/liblzf.rb:25]: Sounds like a good idea

Comment (danielnachun) [reply Formula/liblzf.rb:25]: Yes, actually not only do I need to do that in a `CI-syntax-only` script, but I realized I need to fix the `libxdiff` flat namespace issue separately first and rebase this.

**Comment by danielnachun** [comment]
I upstreamed the flat namespace fix and linked to it in a comment.  I also removed the `conflicts_with` lines and will add them back in a syntax-only PR.

**Review by SMillerDev — APPROVED** [review]

**Pull request closed without merging**